src/main/java/racingcar/domain/Racing.java

```diff
@@ -0,0 +1,60 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Racing {
+    private final static int MOVING_FORWARD = 4;
+    private final static int FIRST_CAR_INDEX = 0;
+    private final static int SECOND_CAR_INDEX = 1;
+
+    public List<Car> toCarList(List<String> carNameList) {
+        List<Car> carList = new ArrayList<>();
+
+        for (String carName : carNameList) {
+            carList.add(new Car(carName));
+        }
+
+        return carList;
+    }
+
+    public void initCarsRandomNum(List<Car> carList) {
+        for (Car car : carList) {
+            car.initRandomNum();
+        }
+    }
+
+    public void modifyForwardNum(List<Car> carList) {
+        carList.stream()
+                .filter(car -> isMove(car))
+                .forEach(car -> car.increaseForwardNum());
```

Comment:
메서드 참조를 사용해보면 어떨까요?? 👍

```suggestion
                .filter(this::isMove)
                .forEach(Car::increaseForwardNum);
```

개인적으로 더 깔끔하다는 생각이 들어서요 🥺